sql/moz-fx-data-shared-prod/firefox_desktop/newtab_live/metadata.yaml

```diff
@@ -6,8 +6,4 @@ description: |-
   `firefox-desktop/newtab`
   document type.
 
-  Clustering fields: `submission_timestamp`
```

Comment:
This file should be updated to add `workgroup:mozsocial/prefect` to the existing `workgroup_access` instead of removing it.